VisualRecognitionV4.playground/Pages/Images.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,81 @@
+//:## Images
+
+import Foundation
+import VisualRecognitionV4
+
+let visualRecognition = setupVisualRecognitionV4()
+
+//:### Add images
+
+visualRecognition.addImages(
+    collectionID: "60b4a98f-2472-4e2b-9c73-28bcaea6fa00",
+    imageURL: ["https://watson-developer-cloud.github.io/doc-tutorial-downloads/visual-recognition/2018-Honda-Fit.jpg"],
+    trainingData: nil
```

Comment:
replace `nil` with `training_data.json`
appears in https://test.cloud.ibm.com/apidocs/visual-recognition/visual-recognition-v4?code=swift#add-images

Reply:
It's not that straightforward for the Swift example. We will need to pass in the expected model `TrainingDataObjects`, which is done in the `Add training data to an image` example. I will need to copy that over to this example.

Reply:
NVM it just takes a string. So still an extra step but not the same as the other method.